### PR TITLE
Proposal to rename the cell functions in Spreadsheet

### PR DESCRIPTION
**Pull request by danielsjf**
Initial change as laid out in #188. Unfortunately it builds upon #187, so until that one is approved, it contains all these changes as well.

### CODE REVIEW

**Comment by brianmay** [comment]
At present time I am having difficulty working out what this change does, that isn't already covered by #187  :-(

**Comment by danielsjf** [comment]
@brianmay, should be better now.

**Review by brianmay — APPROVED** [review]

**Comment by danielsjf** [comment]
@brianmay, I guess this can be merged, unless you want to wait for @vallettea for approval.

**Pull request merged**
